dev/com.ibm.ws.config_fat/.classpath

```diff
@@ -6,18 +6,18 @@
 	<classpathentry kind="src" path="test-applications/configdropins/src"/>
 	<classpathentry kind="src" path="test" output="bin_test"/>
 	<classpathentry kind="src" path="test-applications/mergedconfig/src"/>
-	<classpathentry kind="src" path="test-bundles/test.config.childalias.b/src"/>
+	<classpathentry kind="src" path="test-bundles/test.config.childalias.b/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.